### PR TITLE
Prepare 0.18.2 release

### PR DESCRIPTION
**Pull request by mtreinish**
### Summary

This commit prepares for the 0.18.2 release by bumping the version
strings to reflect the new version. This is just a bugfix release that
fixes bugs since the 0.18.1 release.

### CODE REVIEW

**Comment by mtreinish** [comment]
Marking as on hold as @Cryoris has a critical fix to propose and backport, so waiting in that

**Comment by mtreinish** [comment]
The issue was only on main, so there is nothing to backport. So removing the on hold. Once this merges I will tag the release

**Review by kdk — APPROVED** [review]

**Pull request merged**
